### PR TITLE
PMM-6457 Add API to control PXC via dbaas-controller.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-6457

ref:
- https://github.com/Percona-Lab/pmm-api-tests/pull/125
- https://github.com/percona-platform/dbaas-controller/pull/58
- https://github.com/percona/pmm-managed/pull/468
- https://github.com/percona/pmm/pull/582
- https://github.com/Percona-Lab/pmm-submodules/pull/1124

### CODE REVIEW

File: api/managementpb/dbaas/xtradb_cluster.proto [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+package dbaas.v1beta1;
+
+option go_package = "dbaasv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+// TODO move it to parent directory / package once API is v1-stable
+
+// TODO: find soluton to re-use:
```
Comment (BupycHuk) [inline api/managementpb/dbaas/xtradb_cluster.proto:12]: Should we re-use them?

File: api/managementpb/dbaas/xtradb_cluster.proto [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  ];
+  // ProxySQL container parameters.
+  ProxySQL proxysql = 3 [
+    (validator.field) = {
+      msg_exists: true
+    }
+  ];
+}
+
+// XtraDBCluster service provides public methods for managing XtraDB clusters.
```
Comment (BupycHuk) [inline api/managementpb/dbaas/xtradb_cluster.proto:99]: Usually we put service in the end of the file.

Comment (askomorokhov) [reply api/managementpb/dbaas/xtradb_cluster.proto]: Moved.

File: api/managementpb/dbaas/xtradb_cluster.proto (hunk 1 above)
Comment (askomorokhov) [reply api/managementpb/dbaas/xtradb_cluster.proto]: Nope.

**Review by BupycHuk — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**